scrapers_next/ca/committees.py

```diff
@@ -287,43 +146,43 @@ def process_item(self, item):
             com = ScrapeCommittee(
                 name=comm_name, classification="committee", chamber="upper"
             )
-        com.add_source(self.source.url)
-        com.add_source(comm_url)
-        com.add_link(comm_url, note="homepage")
-        return ChooseType(com, source=URL(comm_url))
+
+        com.add_source(self.source.url, note="Committee List Page")
+        com.add_source(comm_url, note="Committee Detail Page")
+
+        return CommDetails(com, source=URL(comm_url))
 
 
 class AssemblyCommitteeList(HtmlListPage):
```

Comment:
You arent scraping any of the committees here. It looks like only subcommittees are being scraped for the Assembly. I see there are som committees with the same name like Public Safety in the Senate and Assembly but this one isnt listed as a joint committee. I am only getting the 7 subcomittees when I run `poetry run spatula scrape scrapers_next.ca.committees.AssemblyCommitteeList` Can you update the code or address this issue why the Assembly committees are not showing up if that is the desired outcome?

Reply:
Ah yes I see why that looks concerning. It's actually due to [this part of the code](https://github.com/openstates/openstates-scrapers/pull/4397/files#diff-d2282de25dd52de0c420d1d9e3ba2378a39f614b5b6ef2c0590624dbba51f483R164-R167) where all Joint Committees are skipped when running `AssemblyCommitteeList` so as to avoid duplicates (since they are ingested during `SenateCommitteeList()`). That's how it was written before, and I didn't see a reason to change it.

When I did the rewrite I also double-checked that they were definitely all the same committees (i.e. there aren't any joint committees listed on Assembly's page but not on Senate's page, and vice versa), and that is the case. So we're good there.

Reply:
This was bizarre to me, because the CSS selector was grabbing from the top of the list when I ran it last on Friday. Either I've lost my marbles, or the CA site adjusted styling: either of which are very possible.

Anyway, [here is the new selector path](https://github.com/openstates/openstates-scrapers/pull/4397/files/65c52f5361ab0e670b6e6b907154064597359749..5ddda75f5f180d8f90ba20c0841a7117fcb2a8d8#diff-d2282de25dd52de0c420d1d9e3ba2378a39f614b5b6ef2c0590624dbba51f483R157) which gets the whole list. I tested it again and it looks good. Lmk what you get!

Reply:
My bet is CA updated their website, seems like a lot of updates are going on now. I just ran this and the results look good to me. You are correctly scraping the committees as I would expect.